README.md

```diff
@@ -6,10 +6,68 @@
 
 ## Introduction
 
-Frequenz gRPC API for monitoring and control of microgrids.
+The Frequenz Microgrid API is an API for managing and monitoring components
```

Comment:
I would have said "operating and monitoring" but managing is also fine. However, its electrical components and sensors :-)

Reply:
Done